frontend/conf/routes

```diff
@@ -2,8 +2,8 @@
 # This file defines all application routes (Higher priority routes first)
 # ~~~~
 
-GET            /robots.txt                            controllers.CachedAssets.at(path="/public", file="robots.txt")
-GET            /humans.txt                            controllers.CachedAssets.at(path="/public", file="humans.txt")
+GET            /robots.txt                            controllers.CachedAssets.at(path="/public", file="robots.txt", aggressiveCaching:Boolean ?= false )
```

Comment:
It also says:

```
Parameters with fixed values

# Extract the page parameter from the path, or fix the value for /
GET   /                     controllers.Application.show(page = "home")
```

...this just doesn't seem to be working with passing boolean literal true?!
https://www.playframework.com/documentation/2.0.2/ScalaRouting#Parameters-with-fixed-values